### PR TITLE
Mobile small screen issue

### PR DESCRIPTION
**Pull request by mitkoyanev**
As far as I can tell, both landing page, about page, and explore page all scale decently when loaded on small screens.
Known issues: Graphs do not scale well, but are functional and one can scroll through them.
When minimizing left side bar in desktop sizes, the right component all moves with it.

### CODE REVIEW

File: src/components/YearSelector.js [hunk 1]
```diff
@@ -52,7 +53,12 @@ class YearSelector extends Component {
 
     return (
       <div>
-        <p> Select Year of Data to be displayed: </p>
+        {window.innerWidth > mobileStyle ? (
+          <p>Select Year of Data to be displayed:</p>
+        ) : (
+          <p />
```
Comment (steventhan) [inline src/components/YearSelector.js]: Remove the empty `<p>` tag

File: src/components/LeftSideBar.js [hunk 2]
```diff
@@ -169,7 +175,7 @@ class LeftSideBar extends Component {
         );
       } else {
         return (
-          <nav className="scrollLeftBar col-md-2 d-none d-md-block bg-dark sidebar">
+          <nav className="fixed-left scrollLeftBar col-md-2 d-md-block bg-dark sidebar ">
```
Comment (steventhan) [inline src/components/LeftSideBar.js]: Does `fixed-left` do anything? I cant find it anywhere in the code base. Also not sure if display the left side bar on top of the table is a mobile responsive solution. Can you make it slide in and out, maybe?

File: src/styles/LeftSideBar.scss [hunk 3]
```diff
@@ -98,3 +98,9 @@ nav.scrollLeftBar.col-md-2.d-none.d-md-block.bg-light.sidebar {
   width: 30px;
   background-color: #343a40 !important;
 }
+
+.TopSideBarButton {
```
Comment (steventhan) [inline src/styles/LeftSideBar.scss]: `kebab-case` for css naming?

File: src/components/YearSelector.js (hunk 1 above)
Comment (mitkoyanev) [reply src/components/YearSelector.js]: ok

File: src/components/LeftSideBar.js (hunk 2 above)
Comment (mitkoyanev) [reply src/components/LeftSideBar.js]: What, animations?? Uuhhh...haven't made any animations before.. And why not have it fold up? It seems functional on my screen. Where would you recommend the button be instead?

Comment (mitkoyanev) [reply src/components/LeftSideBar.js]: If you press the carrot, it hides the left side bar, just as before.

File: src/styles/LeftSideBar.scss (hunk 3 above)
Comment (mitkoyanev) [reply src/styles/LeftSideBar.scss]: done.

Comment (mitkoyanev) [reply src/styles/LeftSideBar.scss]: done.

File: src/components/YearSelector.js [hunk 4]
```diff
@@ -52,7 +53,12 @@ class YearSelector extends Component {
 
     return (
       <div>
-        <p> Select Year of Data to be displayed: </p>
+        {window.innerWidth > mobileStyle ? (
+          <p>Select Year of Data to be displayed:</p>
+        ) : (
+          {}
```
Comment (steventhan) [inline src/components/YearSelector.js]: This empty block of code should be removed thou, that's what i meant from the previous comment

**Review by steventhan — APPROVED** [review]

**Pull request merged**